maxHeap/src/maxHeap.java

```diff
@@ -0,0 +1,84 @@
+public class maxHeap{
```

Comment:
Class name should always start with Capital letter, Editor should have warned. Please set up your IDE for the same - this should be enough :-D -  https://google.github.io/styleguide/javaguide.html

Reply:
Will keep this in mind from now on. Thanks for the link.